source/manual/prevent-a-page-being-returned-in-search-results.html.md

```diff
@@ -0,0 +1,15 @@
+---
+owner_slack: "#govuk-search"
+title: "Prevent a page being returned in search results"
+parent: "/manual.html"
+layout: manual_layout
+section: Search
+---
+
+## How to filter a page from Site search results
+
+To prevent a page being returned in site search results, a large negative boost can be applied to the DiscoveryEngine serving configuration.
+
+Add the path the filtered_pages array defined in [here][link-1]
```

Comment:
Typo: a "to" is missing between "Add the path **to** the filtered_pages array"

Regarding the link, I'd limit the lines to only the array, and use a unique sha rather than linking to main which will change. E.g. https://github.com/alphagov/govuk-infrastructure/blob/0fbbc7017fd0a6334b38df72731f664252f14dbd/terraform/deployments/search-api-v2/serving_config_global_default.tf#L116-L120